.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Description of the issue**
+
+**How to reproduce the issue**
+
+\`\`\`
+
+put code in code blocks
+
+\`\`\`
+
+\<details\>
+
+put long logs in details blocks
+
+\<\/details\>
+
+**Cirq version (optional)**
+You can get the cirq version by printing `cirq.__version__`. From the command line this can be obtained by running 
```

Comment:
nit: "From the command line:" is shorter, less formal and conveys the same information.

Reply:
done